WIP: Update community info.

Ismael-VC opened the pull request:
Please let me know of any suggestion, thanks in advance!

waldyrious commented on community/index.md:
```diff
@@ -80,21 +87,76 @@ Various Julia videos and playlists are posted on [YouTube](https://www.youtube.c
+## Gitter
+
 * [julia](https://gitter.im/JuliaLang/julia) Gitter chat room is for discussions around the usage of Julia.
-* [julialang-es](https://gitter.im/JuliaLangEs/julialang-es) Gitter chat room is for discussions around the usage of Julia in the Spanish language.
```
Why was this removed?

waldyrious commented on community/index.md:
```diff
@@ -151,9 +229,15 @@ South America
 
-
-[julia-users]: https://groups.google.com/group/julia-users
+Don't see your community resourses listed here? Please submit a [pull request](https://github.com/JuliaLang/julialang.github.com/edit/master/community/index.md) so we can share them!
```
Typo: should be "resources".

Ismael-VC commented on community/index.md:
```diff
@@ -80,21 +87,76 @@ Various Julia videos and playlists are posted on [YouTube](https://www.youtube.c
+* [JuliaOpt](https://gitter.im/JuliaOpt/home)
+* [UCIDataScienceInitiative](https://gitter.im/UCIDataScienceInitiative/home)
+* [Julia-i18n](https://gitter.im/Julia-i18n/julia-i18n) Gitter chat room is for discussions around Julia internationalization and localization.
+* [julialang-es](https://gitter.im/JuliaLangEs/julialang-es) Gitter chat room is for discussions around the usage of Julia in the Spanish language.
```
@waldyrious here is the line you are looking for, notice that also the JuliaLangPt cahnel is now listed below too! 👍

waldyrious replied:
Got it :) the way the diff is displayed (with the removed "QQ groups" heading in the middle of this section) threw me off.

MichaelHatherly commented on community/index.md:
```diff
@@ -80,21 +87,76 @@ Various Julia videos and playlists are posted on [YouTube](https://www.youtube.c
+* [GR.jl](https://gitter.im/jheinen/GR.jl)
+* [FixedSizeArrays.jl](https://gitter.im/SimonDanisch/FixedSizeArrays.jl)
+* [DSGE.jl](https://gitter.im/FRBNY-DSGE/DSGE.jl)
+* [Docile.jl](https://gitter.im/MichaelHatherly/Docile.jl)
```
Docile and Lexicon are both pretty much deprecated on Julia 0.4+ so best to not include those. Rather include the gitter room for the JuliaDocs org https://gitter.im/juliadocs/users in the section above. Could you add the JuliaDocs org to the organisation section as well, thanks @Ismael-VC.

Ismael-VC replied:
Sure thing! I didn't know about that juliadocs chat room.

MichaelHatherly replied:
Hadn't put much effort into publicising it... perhaps I won't need to now :smile:

bicycle1885 commented on community/index.md:
```diff
@@ -80,21 +87,76 @@ Various Julia videos and playlists are posted on [YouTube](https://www.youtube.c
+* [DSGE.jl](https://gitter.im/FRBNY-DSGE/DSGE.jl)
+* [Docile.jl](https://gitter.im/MichaelHatherly/Docile.jl)
+* [Mathematica.jl](https://gitter.im/MikeInnes/Mathematica.jl)
+* [Nanopore.jl](https://gitter.im/gawbul/Nanopore.jl)
```
Nanopore.jl is an empty package and no updates for two years. So, I guess this is abandoned.

Ismael-VC replied:
I've just been searching gitter rooms, haven't checked yet if all those packages are registered and active, but will do.

ChrisRackauckas replied:
You're missing the ArbFloats chatroom. Also, there's a lot of "defunct" rooms here. Most of them have no posts, or no posts in the last year. Maybe any room that hasn't had a post in a year should be dropped? It's just a bad showing to have a bunch of empty rooms.

I don't know what "the rule" should be, but rooms with 0 posts in 2 years? I think it's clear they just clutter the list.

ChrisRackauckas commented on community/index.md:
```diff
@@ -80,21 +87,76 @@ Various Julia videos and playlists are posted on [YouTube](https://www.youtube.c
+* [JuliaOpt](https://gitter.im/JuliaOpt/home)
+* [IntelLabs](https://gitter.im/IntelLabs/home)
+* [JuliaOpt](https://gitter.im/JuliaOpt/home)
+* [UCIDataScienceInitiative](https://gitter.im/UCIDataScienceInitiative/home)
```
You're missing [JuliaDiffEq](https://gitter.im/JuliaDiffEq/Lobby). Also, the data science initiative one doesn't need to be here: it's just for an old workshop.

Ismael-VC replied:
Got it! I had it there in case you planned to repeat the workshop.

oxinabox commented:
Shall we leave the groups listed in #445 for another PR?
(or until owners volunteer them)

Ismael-VC commented:
> (or until owners volunteer them)

@oxinabox what do you mean by *owner volunteer them*?

I think it's ok to add them here either way, I was unaware of # 444 and # 445. It's great that you've already discussed organizing that list.

oxinabox commented:
>@oxinabox what do you mean by until owners volunteer them?

As in, perhaps there is a reason the creators/maintainers of some of those groups have not raised an issue/a PR to have them added.
So maybe we should wait until they do?

KristofferC commented on community/index.md:
```diff
@@ -80,21 +87,76 @@ Various Julia videos and playlists are posted on [YouTube](https://www.youtube.c
+* [Gadfly.jl](https://gitter.im/dcjones/Gadfly.jl)
+* [LightGraphs.jl](https://gitter.im/JuliaGraphs/LightGraphs.jl)
+* [GR.jl](https://gitter.im/jheinen/GR.jl)
+* [FixedSizeArrays.jl](https://gitter.im/SimonDanisch/FixedSizeArrays.jl)
```
AFAIU, FixedSizeArrays is on the deprecation path in favour of StaticArrays.jl so maybe no point in including it here. @SimonDanisch likely has a more informed opinion than me.

SimonDanisch replied:
Yeah, I almost finished transferring all my packages to StaticArrays :) Just a few bugs left, but then the deprecation for FixedSizeArrays in 0.6 should be pretty final!

SimonDanisch commented:
I'd also add `JuliaGeometry` - https://github.com/JuliaGeometry

ChrisRackauckas commented:
Bumping this so it doesn't get lost.

ViralBShah commented:
Still needs to address some of the comments here (at least spellings)

ViralBShah commented:
Bump.

JeffreySarnoff commented:
last paragraph: spell as "resources"

StefanKarpinski commented:
Bump, this should get merged – @Ismael-VC, can you update?

Ismael-VC commented:
Certainly, I'm on it right now.

On Jan 23, 2017 7:31 PM, "Stefan Karpinski" <notifications@github.com>
wrote:

> Bump, this should get merged – @Ismael-VC <https://github.com/Ismael-VC>,
> can you update?
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/JuliaLang/julialang.github.com/pull/460#issuecomment-274674089>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AEYcid4_BW9ytTrbr0kRCvrHMi1OyYS2ks5rVVRqgaJpZM4K_Vyz>
> .
>

Ismael-VC commented:
@JeffreySarnoff

> "Old lists" is misleading (no longer in use?) maybe
> "Domain Resources" or "Collaborative Repositories"

 * I have used "Domain Resources" instead.

@SimonDanisch @KristofferC 

* removed FixedSizeArrays reference
* is there a StaticArrays.jl gitter room?
* added JuliaGeometry

@ChrisRackauckas 

* added JuliaDiffEq

@waldyrious 

* fixed typo

@MichaelHatherly 

* added JuliaDocs

@bicycle1885 

* removed Nanopore

@oxinabox 

* added missing orgs from #445 
* orgs listed based on #444 and #477

Ismael-VC commented:
@ViralBShah @StefanKarpinski I've enhanced, updated and rebased, please let me know if there is anything else that should be fixed or added, thanks in advance and sorry for the wait.

ViralBShah commented:
I think further reorganization will be needed since there is a lot on this page. But merging for now as we can update in future PRs.
